lib/dbus/introspect.rb

```diff
@@ -210,4 +223,39 @@ def to_xml
       xml
     end
   end # class Signal
+
+  # An (exported) property
+  # https://dbus.freedesktop.org/doc/dbus-specification.html#standard-interfaces-properties
+  class Property
+    # @return [String] The name of the property, for example FooBar.
+    attr_reader :name
+    attr_reader :type
+    # @return [Symbol] :read :write or :readwrite
+    attr_reader :access
+
+    # @return [Symbol] What to call at Ruby side. (TODO clarify :foo vs :foo=)
+    attr_reader :ruby_name
+
+    def initialize(name, type, access, ruby_name:)
```

Comment:
well, when you use named param, why not have default `ruby_name: name.to_sym`? so
```suggestion
def initialize(name, type, access, ruby_name: name.to_sym)
```

Reply:
No, a simple `to_sym` conversion is not wanted and I do want to keep the logic out of this class.

Having missed it for methods and signals, for properties I use the correct naming conventions: `foo_bar` at the Ruby side is `FooBar` at the D-Bus side.